test/functional/tests/security/test_security_user.py

```diff
@@ -492,3 +490,9 @@ def run_as_other_user(command, user: str, sudo: bool = False):
     if output.exit_code != 0 or output.stderr is not "":
         raise CmdException("Must be run as root.", output)
     return output
+
+
+def add_user():
+    create_user(user_name, additional_params=["N", "r", "l"])
+    if not check_if_user_exists(user_name):
+        raise Exception("Failed to create user.")
```

Comment:
I suggest reversed implementation ;) if not exists: create_user, and throw inside create_user